app/models/spree/gateway/pay_pal_express.rb

```diff
@@ -1,9 +1,11 @@
 require 'paypal-sdk-merchant'
```

Comment:
Do we still need to require `paypal-sdk-merchant` ?
If not probably we should remove also the dependency in the `.gemspec`

Reply:
@apellizzn some other parts still depend on it (See: https://github.com/mikamai/solidus_paypal_express/pull/1/files#diff-7bc96aa28169887685dfc84a42506992R126) so I wanted to figure out if we can just remove it and ActiveMerchant will take care of the rest or if we need to make some other changes. I'll try and run some regression testing through the Forst app.